### PR TITLE
deploy machine-healthcheck with mao controllers

### PR DESCRIPTION
**Pull request by enxebre**
- Deploy CRDs and machine-healtcheck with mao controllers
- In a follow up let's consider to move all CRDs and manifests but the deployment to the install folder
- Let's wait for https://github.com/openshift/machine-api-operator/pull/118 to merge

### CODE REVIEW

File: owned-manifests/machinehealthcheck.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  name: machinehealthchecks.healthchecking.openshift.io
+spec:
+  group: healthchecking.openshift.io
+  names:
+    kind: MachineHealthCheck
+    listKind: MachineHealthCheckList
+    plural: machinehealthchecks
+    singular: machinehealthcheck
+  scope: Namespaced
+  version: v1alpha1
```
Comment (ingvagabund) [inline owned-manifests/machinehealthcheck.crd.yaml]: Should not we at least validate the metadata? See https://github.com/openshift/machine-api-operator/blob/master/owned-manifests/machine.crd.yaml#L16-L24

File: owned-manifests/clusterapi-manager-cluster-role.yaml [hunk 2]
```diff
@@ -36,3 +36,10 @@ rules:
   - patch
   - delete
 
+- apiGroups:
+  - healthchecking.openshift.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
+s
```
Comment (paulfantom) [inline owned-manifests/clusterapi-manager-cluster-role.yaml]: What's this?

**Pull request merged**